spec/spec_test.rb

```diff
@@ -0,0 +1,24 @@
+require './test.rb'
+
+RSpec.describe Test do
+  it "Find the largest value obtained by permutations of digits." do
+    t = Test.new
```

Comment:
You can also declare the instance once as a variable, so we don't need to repeatedly initiate it kub

```ruby
let(:instance) { described_class.new }

it 'works' do
  expect(instance.large_val(123)).to eq 321
end

```

Reply:
And normally we name a spec file with a `_spec` suffix, so in this case, it should be `test_spec.rb`